tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/ServerlessMiniAgent.cs

```diff
@@ -0,0 +1,165 @@
+// <copyright file="ServerlessMiniAgent.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation;
+
+internal static class ServerlessMiniAgent
+{
+    private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(ServerlessMiniAgent));
+
+    internal static string GetMiniAgentPath(PlatformID os, ImmutableTracerSettings settings)
+    {
+        if (!settings.IsRunningInGCPFunctions && !settings.IsRunningInAzureFunctionsConsumptionPlan)
+        {
+            return null;
+        }
+
+        if (EnvironmentHelpers.GetEnvironmentVariable("DD_MINI_AGENT_PATH") != null)
+        {
+            return EnvironmentHelpers.GetEnvironmentVariable("DD_MINI_AGENT_PATH");
+        }
```

Comment:
We can avoid calling `EnvironmentHelpers.GetEnvironmentVariable()` twice.
```csharp
        var path = EnvironmentHelpers.GetEnvironmentVariable("DD_MINI_AGENT_PATH");

        if (path != null)
        {
            return path;
        }
```

Or (recommended) use the newer [declaration pattern](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/patterns#declaration-and-type-patterns) syntax (where `{ }` means not null):
```suggestion
        if (EnvironmentHelpers.GetEnvironmentVariable("DD_MINI_AGENT_PATH") is { } path)
        {
            return path;
        }
```